auditlog/models/log.py

```diff
@@ -0,0 +1,54 @@
+# -*- coding: utf-8 -*-
+##############################################################################
+#
+#    OpenERP, Open Source Management Solution
+#    Copyright (C) 2013 ABF OSIELL (<http://osiell.com>).
+#
+#    This program is free software: you can redistribute it and/or modify
+#    it under the terms of the GNU Affero General Public License as
+#    published by the Free Software Foundation, either version 3 of the
+#    License, or (at your option) any later version.
+#
+#    This program is distributed in the hope that it will be useful,
+#    but WITHOUT ANY WARRANTY; without even the implied warranty of
+#    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#    GNU Affero General Public License for more details.
+#
+#    You should have received a copy of the GNU Affero General Public License
+#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+##############################################################################
+
+from openerp import models, fields
+
+
+class auditlog_log(models.Model):
+    _name = 'auditlog.log'
+    _description = "Auditlog - Log"
+    _order = "create_date desc"
+
+    name = fields.Char("Resource Name", size=64)
+    model_id = fields.Many2one(
+        'ir.model', string=u"Model")
+    res_id = fields.Integer(u"Resource ID")
+    user_id = fields.Many2one(
+        'res.users', string=u"User")
+    method = fields.Char(u"Method", size=64)
+    line_ids = fields.One2many(
+        'auditlog.log.line', 'log_id', string=u"Fields updated")
+
+
+class auditlog_log_line(models.Model):
+    _name = 'auditlog.log.line'
+    _description = "Auditlog - Log details (fields updated)"
+
+    field_id = fields.Many2one(
+        'ir.model.fields', string=u"Field", required=True)
```

Comment:
Would you consider making this ondelete="cascade"? I'm thinking of migrations between major versions, in which the datamodel can change drastically sometimes.